build/check-style.sh

```diff
@@ -107,7 +107,7 @@ TestReturnCheck() {
 }
 
 TestVet() {
-  ! go tool vet -all -shadow -printfuncs Info:1,Infof:1,InfofDepth:2,Warning:1,Warningf:1,WarningfDepth:2,Error:1,Errorf:1,ErrorfDepth:2,Fatal:1,Fatalf:1,FatalfDepth:2,UnimplementedWithIssueErrorf:1 . 2>&1 | \
```

Comment:
A new commit is submitted. Differences with your suggestion:
- use `local vet` in stead of 'vet'.
- put double quotes around `$vet`. Otherwise, embedded newlines will be deleted.